pulsar-common/src/main/java/com/yahoo/pulsar/common/api/Commands.java

```diff
@@ -534,15 +534,19 @@ public static ByteBuf newConsumerStatsResponse(CommandConsumerStatsResponse.Buil
     private final static ByteBuf cmdPing = serializeWithSize(
             BaseCommand.newBuilder().setType(Type.PING).setPing(CommandPing.getDefaultInstance()));
 
-    static ByteBuf newPing() {
-        return RecyclableDuplicateByteBuf.create(cmdPing).retain();
+    public static ByteBuf newPing() {
+        ByteBuf pingCmdBuf =  RecyclableDuplicateByteBuf.create(cmdPing).retain();
```

Comment:
Well, there is already one buffer.. though is pooled. 

I think this could be done like : 

```java
private final static ByteBuf cmdPing;

static {
    private final static ByteBuf cmdPing;

    static {
        ByteBuf serializedCmdPing = serializeWithSize(
                   BaseCommand.newBuilder()
                       .setType(Type.PING)
                       .setPing(CommandPing.getDefaultInstance()));
        cmdPing = Unpooled.copiedBuffer(serializedCmdPing);
        serializedCmdPing.release();
    }
}
```